new/egeria-app.container.js

```diff
@@ -33,34 +33,13 @@ class EgeriaApp extends PolymerElement {
   constructor() {
     super();
 
-    this.isLoading = true;
+    this.isLoading = false;
+    let path = window.location.pathname.substr(1)
```

Comment:
can you also add ; at the end, it will make a point

Reply:
done